miden-crypto/src/merkle/mmr/error.rs

```diff
@@ -10,6 +10,8 @@ pub enum MmrError {
     PositionNotFound(usize),
     #[error("mmr peaks are invalid: {0}")]
     InvalidPeaks(String),
+    #[error("mmr forest is out of bounds: {0}")]
+    ForestOutOfBounds(String),
```

Comment:
One small nit: this could be more like `ForestOutOfBounds(usize, usize)` so that we don't have to format strings at the callsite.